photo/tests/test_queries/test_collection.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase
+
+from photo.models import Collection
+from photo.schema import schema
+from photo.tests.factories import CollectionFactory, PictureFactory, UserFactory
+from photo.tests.test_queries.graphql_queries import (
+    collections_query_all,
+    collections_query_filter,
+)
+
+
+class CollectionTest(TestCase):
+    def setUp(self):
+        self.batch = 10
+        user = UserFactory(user_profile_picture=True)
+        self.pictures = PictureFactory.create_batch(self.batch, user=user)
+        self.collections = CollectionFactory.create_batch(
+            self.batch, collection_pictures=self.pictures
+        )
+
+    def test_query_all(self):
```

Comment:
since a query returns everything this test should be called test_query_success or test_query_with_data, and there should another test for an empty query, test_query_without_data

Reply:
Applied change
Added tests for test_query_without_data